webpack.config.js

```diff
@@ -0,0 +1,78 @@
+const path = require("path");
+
+const {CleanWebpackPlugin} = require("clean-webpack-plugin");
+const HtmlWebPackPlugin = require("html-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+module.exports = {
+  mode: "development",
+
+  entry: {
+    main: "./front-end/src/index.tsx"
+  },
+
+  output: {
+    filename: "[name].js",
+    path: path.resolve(__dirname, "build")
+  },
+
+  devServer: {
+    open: true,
+    port: 9000
+  },
+
+  module: {
+    rules: [
+      {
+        test: /\.tsx?$/,
+        use: "ts-loader"
+      },
+
+      {
+        test: /\.(jpg|png|svg|gif)$/,
+        loader: "file-loader",
+        options: {
+          name: "[name].[ext]"
+        }
+      },
+
+      {
+        test: /\.(scss|sass)$/,
+        use: [
+          MiniCssExtractPlugin.loader,
+          "css-loader", 
+          "sass-loader"
+        ]
+      },
+
+      {
+        test: /\.(js|jsx)$/,
```

Comment:
```suggestion
        test: /\.(jsx?)$/,
```

To keep it consistent with `test: /\.tsx?$/,`

Reply:
Or the way around if you prefer. Just make it both the same way.